example/documentation/remotes/simple.js

```diff
@@ -17,7 +17,10 @@ function getSecret(callback) {
  * Takes a string and returns an updated string.
  */
 helper.method(transform, {
-  accepts: [{ name: 'str', type: 'string', required: true, description: 'The value to update' }],
+  accepts: [{ name: 'str',
+              type: 'string',
+              required: true,
+              description: 'The value to update' }],
```

Comment:
IIRC, we are using a different formating in loopback - put just the `description` on the new line:

```js
{
  accepts: [{ name: 'str', type: 'string', required: true,
    description: 'The value to update' }],
}
```